roles/common/tasks/main.yml

```diff
@@ -20,6 +20,26 @@
     retries: 5
   when: ansible_distribution == "Ubuntu"
 
+- name: set selinux to permissive on redhat
+  selinux:
+    policy: targeted
+    state: permissive
+  when: ansible_distribution == "RedHat"
+  tags: selinux
+
+- name: Register with RHN
+  redhat_subscription:
+    state: present
+    username: "{{ redhat.user }}"
+    password: "{{ redhat.pass }}"
+  when: ansible_distribution == "RedHat"
+  tags: rhn
+
```

Comment:
Need to attach to a pool before we can enable the repos

- name: Add OSP pool
  command: subscription-manager attach --pool={{ item }}
  when: ansible_distribution == "RedHat"
  with_items:
  - 8a85f98158880f8d01588bde0f7c3c43